users/wanleg/wanleg.c

```diff
@@ -0,0 +1,163 @@
+#include "wanleg.h"
+
+void persistant_default_layer_set(uint16_t default_layer) {
```

Comment:
This isn't actually needed.  In fact, I don't think you're actually using this function